autoload/LanguageClient.vim

```diff
@@ -201,6 +209,14 @@ function! s:inputlist(...) abort
     return l:selection
 endfunction
 
+function! s:selectionUI_funcref(source, sink) abort
+    if g:LanguageClient_selectionUI ==? 'FZF'
```

Comment:
I think you are missing a check for when the user hasn't set `LanguageClient_selectionUI`. As it stands, if that option is not set, but FZF is loaded, it will use the location list anyways.